.github/workflows/CI.yml

```
@@ -20,7 +20,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: '3.13' # Adjust Python version as needed
+        python-version: '3.12' # Adjust Python version as needed
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)